examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    async def acquire_data_async(args):
</pre><p>can you give your arguments comprehensible names. What is the args[0]? an image? a matrix? </p><p> </p><p>also, need to explain what the function acquire_data does and where it comes from</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>